ShortcutsSwift.playgroundbook/Contents/Sources/BuildShortcut.swift

```diff
@@ -39,12 +39,16 @@ public func exportShortcut(_ actionContainer: ActionContainer) -> Data {
     return try! PropertyListSerialization.data(fromPropertyList: buildShortcut(actionContainer), format: .binary, options: 0)
 }
 
+public func exportShortcut(_ shortcut: PropertyList) -> Data {
```

Comment:
This makes it easier to save the shortcut on the mac and send it to the phone for testing.